modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -240,6 +246,18 @@ config TFM_CONNECTION_BASED_SERVICE_API
 	  system. When this option is not enabled in the TF-M build system this
 	  will result in compilation error.
 
+
+if TFM_BL1
+
+config TFM_BL2_SIGNING_KEY_PATH
+	string "Path to private key used to sign BL2 firmware images."
+	help
```

Comment:
shouldn't this have a default then since the help says it does?

Reply:
No, empty means use default one in the tf-m, if user would like to sign it with a different key, then this config can be used over zephyr.